doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md

```diff
@@ -56,6 +69,76 @@ Since Pimcore X supports only Symfony 5, which dropped the support for Php templ
 
 If you are using enterprise edition, then it is still possible to support Php templates by installing `pimcore/php-templating-engine-bundle`.
 
+### Additional Changes for Templates
```

Comment:
I would keep this as a sub section of "Migrate Php templates to Twig" and maybe move RegEx Examples to gist

Reply:
External Link is implemented and is moved to the "Migrate Php templates to Twig" Section